command/e2etest/init_test.go

```diff
@@ -340,7 +340,8 @@ func TestInitProviderNotFound(t *testing.T) {
 			t.Fatal("expected error, got success")
 		}
 
-		if !strings.Contains(stderr, "provider registry\nregistry.terraform.io does not have a provider named\nregistry.terraform.io/hashicorp/nonexist") {
+		oneLineStderr := strings.ReplaceAll(stderr, "\n", " ")
```

Comment:
I suddenly feel silly for not doing something like this in many other tests and will copy this pattern going forward. ⭐